### ISSUE TITLE
Roaring intersect returns wrong result when passing an interval outside the range of the bitmap

### ISSUE THREAD

**Issue by mdouaihy** [RoaringBitmap/RoaringBitmap#333]
The following test fails. roaringBitmap.intersects(12, 14) returns true.
```
@Test
   public void testIntersect() {
      RoaringBitmap roaringBitmap = new RoaringBitmap();
      roaringBitmap.add(0L, 3);
      roaringBitmap.add(6L, 10);
      Assert.assertFalse(roaringBitmap.intersects(12, 14));
   }
```

The error seems to be due to the last instruction in method intersects
highLowContainer.getContainerAtIndex(pos)
                               .intersects(0, (int)((supremum - 1) & 0xFFFF) + 1)

where the minimum is forced to 0.

**Comment by mdouaihy** [comment]
Thank you @lemire  and @MagalieElOsta.

Do you think we can have a new patched release containing this fix soon?

**Comment by lemire** [comment]
This issue is not verified as being completely fixed yet.

**Comment by lemire** [comment]
https://github.com/RoaringBitmap/RoaringBitmap/pull/335

**Comment by lemire** [comment]
Hopefully now fixed with https://github.com/RoaringBitmap/RoaringBitmap/pull/335

Closing.

**Comment by lemire** [comment]
Please see
https://github.com/RoaringBitmap/RoaringBitmap/issues/340

**Comment by lemire** [comment]
Fixed in RoaringBitmap-0.8.5.
